metricproviders/datadog/datadog_test.go

```diff
@@ -25,6 +25,19 @@ func TestRunSuite(t *testing.T) {
 
 	unixNow = func() int64 { return 1599076435 }
 
+	ddProviderIntervalDefault := v1alpha1.MetricProvider{
```

Comment:
sonarcloud failed with 40.5% duplication in this file, so I only did this part to tidy it all up